src/pages/onboarding/OnboardingScreen.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { View } from 'react-native'
+import AppIntroSlider from 'react-native-app-intro-slider'
+
+import styles from './style'
+import { OnBoardingData } from '../../data/data'
+import OnboardingButtons from './components/Buttons/OnBoardingButtons'
+import OnBoardingRenderItem from './components/renderItem/OnBoardingRenderItem'
+
+class IntroSlider extends React.Component {
```

Comment:
Remove the IntroSlider class and use the below code inside the OnBoarding class

<View style={styles.background}>
    <AppIntroSlider data={OnBoardingData} renderItem={({ item }) => <OnBoardingRenderItem item={item} />} />
    <OnboardingButtons navigation={navigation} />
</View>